src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperRunner.cs

```diff
@@ -22,6 +23,7 @@ public class TagHelperRunner
         {
             var tagHelperContext = new TagHelperContext(
                 executionContext.AllAttributes,
+                executionContext.Items,
```

Comment:
should confirm `TagHelperRunner` sets up `TagHelperContext` as expected in `TagHelperRunnerTest`